types/react-native-video/react-native-video-tests.tsx

```diff
@@ -54,8 +54,9 @@ class VideoTest extends React.Component {
     onProgress = (data: {
         currentTime: number,
         playableDuration: number,
+        seekableDuration: number
```

Comment:
The test would actually have passed without the change to `index.d.ts`! Fixed in #29583.